modules/features2d/test/test_affine_feature.cpp

```diff
@@ -82,7 +83,7 @@ TEST(Features2d_AFFINE_FEATURE, regression)
     const float badCountsRatio = 0.01f;
     const float badDescriptorDist = 1.0f;
     const float maxBadKeypointsRatio = 0.15f;
-    const float maxBadDescriptorRatio = 0.15f;
+    const float maxBadDescriptorRatio = 0.41f;
```

Comment:
It is changed because this test calls `warpAffine` on `8UC1` input and the new kernel for `8UC1` uses FP16 intrinsics which is less accurate than FP32 version. BTW, FP32 version is fine with the original threshold.